README.md

```diff
@@ -1,43 +1,55 @@
 # Edvart
 
-Exploratory Data Analysis (EDA) is a very initial task a data scientist
-or data analyst does when he reaches new data.
-EDA refers to the critical process of performing
-initial investigations on data to discover patterns, to spot
-anomalies, to test hypothesis and to check assumptions with the help
-of summary statistics and graphical representations.
+Exploratory Data Analysis (EDA) is the initial task a data scientist or data
+analyst undertakes when they obtains new data. EDA refers to the critical
```

Comment:
*a data scientist ... they obtains* does not seem correct.
I would either make the whole sentence plural or change *they obtains* -> *upon obtaining* or similar.